Modules/DelayNetworkedData.cs

```diff
@@ -76,6 +77,110 @@ public static void DelaySpawnPlayerInfo(InnerNetClient __instance, int clientId)
+    public static bool SendAllStreamedObjectsPrefix(InnerNetClient __instance, ref bool __result)
+    {
+        if (!Constants.IsVersionModded() || __instance.NetworkMode != NetworkModes.OnlineGame) return true;
+        // Bypass all NetworkedData here.
+        __result = false;
+        Il2CppSystem.Collections.Generic.List<InnerNetObject> obj = __instance.allObjects;
+        lock (obj)
+        {
+            for (int i = 0; i < __instance.allObjects.Count; i++)
+            {
+                InnerNetObject innerNetObject = __instance.allObjects[i];
+                if (innerNetObject && innerNetObject is not NetworkedPlayerInfo && innerNetObject.IsDirty && (innerNetObject.AmOwner || (innerNetObject.OwnerId == -2 && __instance.AmHost)))
+                {
+                    MessageWriter messageWriter = __instance.Streams[(int)innerNetObject.sendMode];
+                    messageWriter.StartMessage(1);
+                    messageWriter.WritePacked(innerNetObject.NetId);
+                    try
+                    {
+                        if (innerNetObject.Serialize(messageWriter, false))
+                        {
+                            messageWriter.EndMessage();
+                        }
+                        else
+                        {
+                            messageWriter.CancelMessage();
+                        }
+                        if (innerNetObject.Chunked && innerNetObject.IsDirty)
+                        {
+                            __result = true;
+                        }
+                    }
+                    catch (Exception ex)
+                    {
+                        Logger.Exception(ex, "SendAllStreamedObjectsPrefix");
+                        messageWriter.CancelMessage();
+                    }
+                }
+            }
+        }
+        for (int j = 0; j < __instance.Streams.Length; j++)
+        {
+            MessageWriter messageWriter2 = __instance.Streams[j];
+            if (messageWriter2.HasBytes(7))
+            {
+                messageWriter2.EndMessage();
+                __instance.SendOrDisconnect(messageWriter2);
+                messageWriter2.Clear((SendOption)j);
+                messageWriter2.StartMessage(5);
+                messageWriter2.Write(__instance.GameId);
+            }
+        }
+        return false;
+    }
+
+    private static byte timer = 0;
+    [HarmonyPatch(typeof(InnerNetClient), nameof(InnerNetClient.FixedUpdate))]
+    [HarmonyPostfix]
+    public static void FixedUpdatePostfix(InnerNetClient __instance)
+    {
+        // Send a networked data pre 2 fixed update should be a good practice?
+        if (!__instance.AmHost || __instance.Streams == null || __instance.NetworkMode != NetworkModes.OnlineGame) return;
+
+        if (timer == 0)
+        {
+            timer = 1;
+            return;
+        }
+
+        var player = GameData.Instance.AllPlayers.ToArray().Where(x => x.IsDirty).FirstOrDefault();
```

Comment:
You can use
`.FirstOrDefault(x => x.IsDirty);`

Since `FirstOrDefault()` or `Any()` and ect. also contains `Where()`

Reply:
I'll change it myself